docs/guide/content_rendering.md

```diff
@@ -422,140 +399,124 @@ You can use this controller from templates with the following syntax:
 
 The example above renders the Content item whose ID is **123** with the view type **line**.
 
-Reference of `ez_content` controller follows the syntax of *controllers as a service*, [as explained in Symfony documentation](http://symfony.com/doc/current/cookbook/controller/service.html).
+Referencing the `ez_content` controller follows the syntax of *controllers as a service*, [as explained in Symfony documentation](http://symfony.com/doc/current/cookbook/controller/service.html).
 
 ##### Available arguments
 
 As with any controller, you can pass arguments to `ez_content:viewLocation` or `ez_content:viewContent` to fit your needs.
```

Comment:
seems to still be here, wasn't this deprecated?

Reply:
AFAIK it was, but changing this is doc was separeted into https://jira.ez.no/browse/EZP-28086, because it turned out to be a larger issue that will require more information from devs.